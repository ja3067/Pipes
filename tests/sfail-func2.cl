foo : str = "hello"

def foo(x):
	return x
