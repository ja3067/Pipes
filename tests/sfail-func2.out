STypeError: expression of type func cannot be assigned to variable 'foo' with explicit type str
